### ISSUE TITLE
Player nickname randomly changed to {nickname}

### ISSUE THREAD

**Issue by Blackdigits** [Zrips/CMI#234]
Description of issue or feature request: Player's nickname are randomly turned into {nickName}, and I have no idea why. It only happened after the update

#### IGNORE IF YOU ARE SUBMITTING A FEATURE REQUEST ####

ERROR:
No error on logs

Cmi Version (using`/cmi version`): 7.6.5.1
Server Type (Spigot/Paperspigot/etc):  Paperspigot
Server Version (using `/ver`):  1373

**Comment by MCTyler** [comment]
Cannot reproduce....
```
Cmi Version (using/cmi version): 7.6.5.2
Server Type (Spigot/Paperspigot/etc): PaperClip
Server Version (using /ver): git-Paper-"344a125" (MC: 1.12.2)
Vault: 1.6.9
```

**Comment by LogGits** [comment]
Fixed in 7.6.5.2. Please update your build of cmi.
